lesson_02/ht_template/job2/tests/tests_dal/test_read_sales.py

```diff
@@ -0,0 +1,22 @@
+from unittest import TestCase, mock
+from lesson_02.ht_template.job2.dal.read_sales import get_sales_from_file
+import json
+
+
+SAMPLE_DATA = [{"client": "Michael Wilkerson", "purchase_date": "2022-08-10",
+                "product": "Vacuum cleaner", "price": 346}]
+
+
+class TestReadSales(TestCase):
+    """
+    tests read_sales function
+    """
+    def test_read_sales_success(self):
+        mock_open = mock.mock_open(read_data=json.dumps(SAMPLE_DATA))
```

Comment:
Another way is to write test data into a temporary directory in setUp.
But this works as well.